engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -41,6 +41,11 @@ public class RbacRecipientUsersProvider {
 
     public static final String ORG_ADMIN_PERMISSION = "admin:org:all";
 
+    public static final String USE_ORG_ID = "notifications.use-org-id";
```

Comment:
Is this the same config used in the other PR? Maybe we could put it in a single place to prevent any surprise.

Reply:
yes, that's the same. Thought about it as well and decided not to do this as I will have to remove the "notifications.use-org-id" from the application.properties anywy and then I will also remove each accountId and these entries. 

I think it's not needed in this step to be honest.